scripts/build.js

```diff
@@ -20,59 +20,93 @@ function escapeForRegExp(str) {
   return str.replace(/[|\\*?+$^.{}()\[\]]/g, "\\$&");
 }
 
-const extensionPath = path.resolve(
-  __dirname,
-  "../continue/extensions/vscode/src",
-);
-const extensionFilter = new RegExp(
-  "^" + escapeForRegExp(extensionPath + path.sep) + ".*\\.ts$",
-);
+const makeTransformPlugin = ({ name, basePath, substitutions }) => {
```

Comment:
it's a minor thing, but the diffs might be slightly clearer to read if there were two commits:

1. build: Factor out transform plugin construction to new makeTransformPlugin function in build script
2. build: Use makeTransformPlugin to transform EXTENSION_NAME and control plain info

or similar. this is just take-or-leave it advice, opinions may differ, etc etc...

Reply:
True, commit split.